### DISCUSSION TITLE
Error message

### DISCUSSION THREAD

**Discussion by matpol84** [GorvGoyl/ChatGPT-Writer#36] [Report Bugs and Issues]
I've updated Chrome and re-installed the extension. I still get this error: chat.openai.com needs to review the security of your connection before proceeding. Please visit [chat.openai.com](https://chat.openai.com/) to pass Cloudflare security check and come back.
If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues).

https://vimeo.com/793102515/e2d219b6c0
![2023-01-26_14-26-36](https://user-images.githubusercontent.com/35378563/214931536-966f6607-ab93-42ac-bf20-c00ea37b84e2.jpg)

**Comment by galileu15** [comment]
same here

**Comment by GorvGoyl** [comment]
@galileu15 @matpol84  Please click on **generate reply** button and error will go away.

**Reply by prasadkarnik** [reply]
The issue persists after clicking on "Generate Response".

**Reply by Santhosh-Thanikachalam** [reply]
Yes for me as well, I also tried tweaking the page access setting and relogging into the main chat gpt software.

**Comment by GorvGoyl** [comment]
fixed in version v2.1.4.. please wait for it to go live on chrome extension store https://chrome.google.com/webstore/detail/chatgpt-writer-email-writ/pdnenlnelpdomajfejgapbdpmjkfpjkp/
